### DISCUSSION TITLE
Why two branches mod support and master?

### DISCUSSION THREAD

**Discussion by ivan-kulikov-dev** [OpenViva/OpenViva#19] [Q&A] [answered]
Why two branches: mod support and master?

**Comment by funmaker** [comment, marked as answer]
Mod support branch is based on 0.9 Viva with some fixes and improvements, but it is not being actively developed.
Current master is based off 0.8 Viva and this is the version we will move forward with. We decided that it will be easier to restore original features using this version as a base as 0.9/mod-support has been mostly rewritten and is like a different game at this point. 0.8 is closer to what viva originally was meant to be.
